python/ray/ray_constants.py

```diff
@@ -58,12 +62,14 @@ def env_bool(key, default):
 PROMETHEUS_SERVICE_DISCOVERY_FILE = "prom_metrics_service_discovery.json"
 # Default resource requirements for actors when no resource requirements are
 # specified.
-DEFAULT_ACTOR_METHOD_CPU_SIMPLE = 1
-DEFAULT_ACTOR_CREATION_CPU_SIMPLE = 0
+DEFAULT_ACTOR_METHOD_CPU_SIMPLE = 0 if gcs_actor_scheduling_enabled() else 1
+DEFAULT_ACTOR_CREATION_CPU_SIMPLE = 0.01 if gcs_actor_scheduling_enabled(
+) else 0
```

Comment:
We use `0.01` as the default CPU resource (for actors and normal tasks) because CPU has to be accounted when selecting node for scheduling. Using `1` as the default one is very likely to exclude feasible nodes.

This has been an Ant internal config. Do you have any suggestions? @iycheng

Reply:
BTW, using this config would fail some tests (if they depend on the original default value).

Reply:
@ericl  do you have any suggestions about this one?

Reply:
We can't change these values in minor Ray versions since it would break user code. What prevents GCS actor scheduling from being enabled with the old values?

Reply:
@ericl The old default value was too large. Actors with that default CPU resource could easily exhaust the `cluster_scheduling_resources`'s total CPU resources, because they claim much more CPU than what they really need. When this happens, no feasible would be selected for any following actors.

Reply:
One quick workaround is making the node selector does not take CPU requirement into account.